cmd/node/main.go

```diff
@@ -178,9 +178,8 @@ VERSION:
 		Value: math.MaxUint32,
 	}
 
-	wd, _ = os.Getwd()
-
 	// workingDirectory defines a flag for the path for the working directory.
+	wd, err          = os.Getwd()
```

Comment:
No no, should have a nil var here and assign it in a function:
```GO
var wdDir = ""

//..........somewhere in a func

wdDir, err := os.Getwd()
// do something with err
```

Reply:
moved it to main